libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4507,6 +4508,30 @@ void GCS_MAVLINK::send_attitude() const
         omega.z);
 }
 
+void GCS_MAVLINK::send_attitude_quaternion() const
+{
+    const AP_AHRS &ahrs = AP::ahrs();
+    Quaternion quat;
+    if (!ahrs.get_secondary_quaternion(quat)) {
+        return;
+    }
+    const float repr_offseq_q[] {0,0,0,0};  // unused, but probably should correspond to the AHRS view?
```

Comment:
I guess we could move this down to line 4531 so we just create and send an empty quaternion and then put a comment beside it saying something like "// unused" or something like that maybe..

Reply:
I'm not sure you can create it anonymously inline?